examples/app-npm-webpack/messages/ru.json

```diff
@@ -0,0 +1,18 @@
+{
+	"ru": {
+		"intro-1": "Используйте Globalize для интернационализиции вашего приложения.",
+		"number-label": "Число",
+		"currency-label": "Валюта",
+		"date-label": "Дата",
+		"relative-time-label": "Относительное время",
+		"message-1": "Пример сообщения со смешанными числами \"{number}\", валютой \"{currency}\", датами \"{date}\" и относительным временем \"{relativeTime}\".",
+		"message-2": [
+			"Пример сообщения с поддержкой множественного числа:",
+			"{count, plural,",
+			"   one {У вас осталась одна задача}",
+			"   many {У вас осталось # задач}",
```

Comment:
@markelog, technically you are correct `other` would pick it up.

I tend to agree with @sompylasar that it's good to make all `ru` forms explicit: i.e., `one`, `few`, `many`, `other` for the clarify of the demo.